### ISSUE TITLE
Ny bug hunt på høsten i august?

### ISSUE THREAD

**Issue by cbrevik** [rust-trondheim/.github#7]

**Comment by eirikblekesaune** [comment]
Ja dette er en lur ide. Kanskje den rett og slett skal hete January Bug Hunt som i 2023? Vi lærte jo sist at vi hadde mye på blokka som vi ikke kom igjennom, så vi kan jo ta en eller to bugs hver som vi kan arbeide med som sist. Hva tenker dere om det?

**Comment by cbrevik** [comment]
Ja, tror vi kan jobbe videre på det vi hadde sist. Og tror ikke det er krise om vi gjentar noe av det heller, går ut ifra at det er noe nytt publikum, og for de som var med sist så tåler de fint noe repeat. Har kanskje godt av noe repeat for lærdommen?

**Comment by cbrevik** [comment]
Spm om bug hunt i januar, eller om vi skal ta wasm først. Hva tror du @AndersNS?
